notary-commons/src/main/kotlin/sidechain/iroha/util/ModelUtil.kt

```diff
@@ -69,18 +70,25 @@ object ModelUtil {
      * @param accountId - account to set details
      * @param key - key of detail
      * @param value - value of detail
+     * @param quorum - tx quorum. -1 by default
```

Comment:
Why not to set default value 1 and get rid of that `if`

Reply:
I don't know if it's a good idea or not. Every transaction will contain `setQuorum(1)` then.

Reply:
It's clearer and not more complicated than `if` I think